_rules/aria-presentational-role-no-global-states-properties-p8g918.md

```diff
@@ -0,0 +1,175 @@
+---
```

Comment:
I don't think you should have more than one new rule in a PR.

Reply:
I apologize for this. I based my work on this pattern: https://github.com/act-rules/act-rules.github.io/pull/2167. These three rules are intended to fulfil the "Presentational Roles Conflict Resolution" topic from an author's perspective.